.travis.yml

```diff
@@ -9,6 +9,10 @@ osx_image: xcode9.3
 install:
   - export PREBUILD_SLUG=$TRAVIS_REPO_SLUG
   - export PKG_CONFIG_PATH="/usr/local/opt/libffi/lib/pkgconfig" #node-gfx/node-canvas-prebuilt#76
+  - >-
+    source ~/.nvm/nvm.sh
+    nvm use 10 || { nvm install 10; nvm use 10; }
+    npm install -g node-gyp
```

Comment:
This looks like it was moved from the old `node_version.sh` file, could you do the `nvm` commands in `ci/linux|osx/preinstall.sh`? At that point NVM is already setup.

I think you would just add

```bash
nvm install node # latest
nvm use node # latest
npm install -g node-gyp 
```

Could do Windows node installation in `win/preinstall.sh` for consistency, or leave it in appveyor.yml since it was already there.